probe-rs/src/session.rs

```diff
@@ -267,37 +275,35 @@ impl Session {
 
         let mut components = Vec::new();
 
-        for ap_index in 0..(interface.num_access_ports() as u8) {
+        // TODO
+        let dp = DpAddress::Default;
+
+        // TODO unwrap
+        for ap_index in 0..(interface.num_access_ports(dp).unwrap() as u8) {
```

Comment:
I think we should propagate this error upwards.

But surely we don't want the unwrap :)

Reply:
done